ui/types/ember-data/types/registries/adapter.d.ts

```diff
@@ -10,5 +10,6 @@ import ModelRegistry from 'ember-data/types/registries/model';
  * Catch-all for ember-data.
  */
 export default interface AdapterRegistry {
+  'pki/issuer': PkiIssuerAdapter;
```

Comment:
You'll need to import this to the file too I think. It should have thrown a linting error that `PkiIssuerAdapter` is not defined 🤔

Reply:
Good catch! I think the reason why I didn't get a linting error is that I was still importing it in the component file. I just updated it so that it is being imported into the adapter registry instead 🙂